### PR TITLE
FAT-2672/C350654

### PR DESCRIPTION
**Pull request by re-jas**
[Implement e-2-e automation of test case C350654: Verify automated patron block 'Maximum number of overdue items' removed after overdue item renewed](https://issues.folio.org/browse/FAT-2672)

### CODE REVIEW

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-renewed.cy.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+import UsersOwners from '../../../../support/fragments/settings/users/usersOwners';
+import PaymentMethods from '../../../../support/fragments/settings/users/paymentMethods';
+import LoanPolicy from '../../../../support/fragments/circulation/loan-policy';
+import Conditions from '../../../../support/fragments/settings/users/conditions';
+import Limits from '../../../../support/fragments/settings/users/limits';
+import UsersSearchPane from '../../../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../../../support/fragments/users/usersCard';
+import UserLoans from '../../../../support/fragments/users/loans/userLoans';
+
+function generateUniqueItemBarcodeWithShift(index = 0) {
```
Comment (MaksimTerekhov) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-renewed.cy.js:29]: may be better to move this function into support/utils
It seems like reusable and valuebale function

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-item-renewed.cy.js [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+      cy.wait(120000);
+
+      cy.visit(TopMenu.usersPath);
+      UsersSearchPane.waitLoading();
+      UsersSearchPane.searchByKeywords(userData.barcode);
+      UsersCard.waitLoading();
+      Users.checkIsPatronBlocked(blockMessage, 'Borrowing, Renewals, Requests');
+
+      const itemForCheckIn = itemsData.itemsWithSeparateInstance[0];
+      UsersCard.openLoans();
```
Comment (penfriend) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-item-renewed.cy.js:241]: The test is failing in this method sometimes. Please update the waiter in this method:  cy.wait('@getLoans', { requestTimeout: 10000 });

**Review by penfriend — CHANGES_REQUESTED** [review]

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-renewed.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-renewed.cy.js]: done

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-item-renewed.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-items-overdue-removed-after-item-renewed.cy.js:241]: done

**Review by penfriend — APPROVED** [review]

**Pull request merged**